src/app/admin/admin-config.js

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
If you're talking about stars http://code.mycila.com/license-maven-plugin/#license-templates and https://github.com/codenvy/codenvy-dev-resources/blob/master/codenvy-eclipse-license-resource-bundle/src/main/resources/javadoc.xml
JS have its own template